crates/biome_module_graph/tests/snapshots/test_resolve_export_types.snap

```diff
@@ -42,19 +41,13 @@ export const superComputer = new DeepThought();
 Exports {
   "superComputer" => {
     ExportOwnExport => JsOwnExport(
-      Object {
-        prototype: "DeepThought" {
-          extends: none
-          type_args: []
-        }
-        members: {}
-      }
+      Module TypeId(4)
       Local name: superComputer
     )
   }
   "theAnswer" => {
     ExportOwnExport => JsOwnExport(
-      value: 42
+      Module TypeId(0)
```

Comment:
So `42` is now a type?

Reply:
It was already :) TypeScript's type system supports literals as types too. What you really see that's changed here is that the snapshot now shows the reference to the type instead of the type itself.